app.js

```diff
@@ -0,0 +1,44 @@
+// Node modules
+const bodyParser   = require('body-parser');
+const cookieParser = require('cookie-parser');
+const express	   = require('express');
+const session	   = require('express-session');
+// Custom Middleware
+const passport	   = require('./auth/');
+const configRoutes = require('./routes');
+const exhbs		   = require('./views');
+
+let app = express();
+// Log incoming requests
+app.use((req, res, next) => {
+	console.log(`${req.method} ${req.path}`);
+	next();
+});
+// Parse cookies and POST body
+app.use(cookieParser());
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+// Enable Cookie session
+app.use(session({
+	secret: 'keyboard cat',
```

Comment:
I do not know what the non-default is. Could you elaborate or change it and push changes?